pkg/configuration/configuration.go

```diff
@@ -112,37 +123,54 @@ const (
 	// varTwilioAccountSID is the constant used to read the configuration parameter for the
 	// Twilio account identifier, used for sending SMS verification codes.  Twilio is a service that
 	// provides an API for sending SMS messages anywhere in the world.  http://twilio.com
-	varTwilioAccountSID = "twilio.account_sid"
+	varTwilioAccountSID = "twilio.account.sid"
 
 	// varTwilioAuthToken is the constant used to read the configuration parameter for the
 	// Twilio authentication token, used for sending SMS verification codes
-	varTwilioAuthToken = "twilio.auth_token"
+	varTwilioAuthToken = "twilio.auth.token"
 )
 
 // Registry encapsulates the Viper configuration registry which stores the
 // configuration data in-memory.
 type Registry struct {
-	v *viper.Viper
+	v            *viper.Viper
+	secretValues map[string]string
 }
 
 // CreateEmptyRegistry creates an initial, empty registry.
-func CreateEmptyRegistry() *Registry {
+func CreateEmptyRegistry(cl client.Client) (*Registry, error) {
+
+	err := configuration.LoadFromConfigMap(EnvPrefix, "REGISTRATION_SERVICE_CONFIG_MAP_NAME", cl)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Why loading from another CM? We already use one CM that is called `registration-service`, there is no point in introducing another one.

Reply:
this is for someone else to override the configurations which we set by default or via the dev.yaml file. we can support someone creating their own configmap

Reply:
Ok, I was missing this context. Anyway, please see my comment here: https://github.com/codeready-toolchain/registration-service/pull/122#discussion_r472997398